### PR TITLE
A128KW, A192KW and A256KW encryption and decryption was added

### PR DESCRIPTION
**Pull request by ramunasjurgilas**
An algorithm for symmetric JWE key (A128KW, A192KW and A256KW) encryption and decryption was added. Marius (@mtamu) is the owner of this implementation.

### CODE REVIEW

**Comment by nathan-mohemian** [comment]
Wow, thanks @ramunasjurgilas and @mtamu for the already quite clean-looking PR!

Great that it already has a changelog entry and tests even 😍

As this is a bit of a larger PR it might take a bit in order for us to get time to look through it, I'll keep you guys updated.

**Comment by edvinasgestautas** [comment]
any update on this ?

File: JOSESwift.podspec [hunk 1]
```diff
@@ -3,13 +3,13 @@ Pod::Spec.new do |s|
...
-  s.authors           = { "Daniel Egger" => "daniel.egger@airsidemobile.com", "Carol Capek" => "carol.capek@airsidemobile.com", "Christoph Gigi Fuchs" => "christoph.fuchs@airsidemobile.com" }
+  s.authors           = { "Daniel Egger" => "daniel.egger@airsidemobile.com", "Carol Capek" => "carol.capek@airsidemobile.com", "Christoph Gigi Fuchs" => "christoph.fuchs@airsidemobile.com", "Ramunas Jurgilas" => "rjur@danskebank.lt", "Marius Tamulis" => "mtamu@danskebank.lt" }
   s.homepage          = "https://github.com/airsidemobile/JOSESwift"
   s.documentation_url = "https://github.com/airsidemobile/JOSESwift/wiki"
   s.social_media_url  = "https://twitter.com/airsideout"
 
   s.swift_version     = "5.0"
   s.platform          = :ios, "10.0"
-  s.source            = { :git => "https://github.com/airsidemobile/JOSESwift.git", :tag => "#{s.version}" }
+  s.source            = { :git => "https://github.com/mtamu/JOSESwift.git", :tag => "#{s.version}" }
```
[the remark this answers is not in the document]
Comment (tamulis) [reply JOSESwift.podspec]: Please do, sorry for this leftover.

File: JOSESwift.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -773,7 +773,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = "$(inherited)";
-				DEVELOPMENT_TEAM = 4857DE3DAC;
+				DEVELOPMENT_TEAM = QG88PL75JK;
```
[the remark this answers is not in the document]
Comment (tamulis) [reply JOSESwift.xcodeproj/project.pbxproj]: None seems a great idea 💡

**Comment by tamulis** [comment]
Thank you @daniel-mohemian for the appreciation, it is a pleasure to contribute!

**Comment by ramunasjurgilas** [comment]
I am glad, that I could contribute. :)

**Comment by daniel-mohemian** [comment]
Hi @tamulis and @ramunasjurgilas! We just released [2.1.0](https://github.com/airsidemobile/JOSESwift/releases/tag/2.1.0) including the A128KW, A192KW and A256KW algorithms. Thanks again for your contribution and your patience. 😄

**Comment by ramunasjurgilas** [comment]
Thanks @daniel-mohemian ! ;)

**Pull request merged**